### DISCUSSION TITLE
How to restrict Remote Schemas GraphQL from showing endpoints that we don't need?

### DISCUSSION THREAD

**Discussion by antonioberetini** [hasura/graphql-engine#7157] [Question]
We added a GraphQL endpoint to Remote Schema then we created a relationship within a table to corelate based on a unique identifier. Once this was finished, we see ALL nodes available under the remote schema GraphQL endpoint, under Hasura API tab.  We really need just the one we selected under the relationship settings. Is there a way to only allow the one we need?

Remote Schema Relationship
![image](https://user-images.githubusercontent.com/65619488/124021439-2c439f00-d9b1-11eb-88ec-23d4fda51f96.png)

API explorer:

![image](https://user-images.githubusercontent.com/65619488/124021538-45e4e680-d9b1-11eb-94ee-fc7bdcab2daf.png)

Brand section is coming from remote schema. See previous attachment, we haven't selected that.

**Comment by dsandip** [comment]
@antonioberetini Remote schema permissions is currently the only way to restrict other fields from showing up. You'll have to use this role in graphiQL as well to avoid seeing unwanted nodes. 

@tirumaraiselvan Can perhaps comment about the schema customisation options planned in the near future.

**Reply by antonioberetini** [reply]
Thanks, it looks like that feature is not enabled on our account. 

Also, GraphQL endpoint we are connecting to is not responding in a standard GQL format Data is wrapped in a response not data tag. Is there a way to corelate?

Here is an a sample response:

{
"Time": "00:00:00.77",
"Status": "OK",
"Server": "NHSDEVAPI4",
"Result": {
"searchCommunities": [
{
"name": "Stagecoach Crossing",
"commStatus": "N",
"city": "Kyle",
"zip": "78640"
}
]
}
